ssl/statem/extensions_srvr.c

```diff
@@ -753,6 +766,32 @@ int tls_parse_ctos_psk(SSL *s, PACKET *pkt, unsigned int context, X509 *x,
     }
 
     sess->ext.tick_identity = id;
+
+    now = (uint32_t)time(NULL);
+    agesec = now - (uint32_t)sess->time;
```

Comment:
Will this code be used in ~22 years? Should 64-bit values be used instead?
(Of course I'm not a fan of multiplication and division in packet processing.)